### DISCUSSION TITLE
articles/good-translations

### DISCUSSION THREAD

**Discussion by giscus** [hutusi/hutusi.github.com#165] [Comments]
# articles/good-translations

最近读了《打开：周濂的100堂西方哲学课》，对西方哲学产生了兴趣，于是准备找一本《西方哲学史》来学习。这一找可不要紧，搜出来了多种版本的《西方哲学史》，其作者有：罗素、弗兰克·梯利、希尔贝克、斯通普夫、施杜里希等。另外，每本又有不同的译本：商务印书馆、上海译文、吉林出版社、天津出版社…我到豆瓣的「不同译本讨论」小组中提问，又有网友回复说：“以上都不推荐，推荐劳黑德《哲学的历程》。”这可让选择困难症的我纠结地不行。

https://hutusi.com/articles/good-translations

**Comment by semyin** [comment]
我感觉我只要一读国外的小说，我就头晕，人名好难记啊

**Reply by hutusi** [reply]
特别是一些写家族的小说，名字更难记，往往是孙子和爷爷的名字一样，甚至三代人名字都一样。像《百年孤独》，几代里面男性名称几乎都是“奥雷里亚诺”，看得都快疯了😂

**Reply by semyin** [reply]
😂有些语句还比较晦涩难懂，我经常看着看着就晕了，又转回去看上面说了啥
